scripts/Player.cs

```diff
@@ -0,0 +1,39 @@
+using Godot;
+using System;
+
+public partial class Player : StaticBody2D
+{
+	
+	public float winHeight; // GetViewportRect().Size.Y; wasnt working with integer... idk why
```

Comment:
GetViewportRect.size is a vector2 which stores floats. Why GetViewportSize returns a Rect2 instead of a Rect2i (the integer variant) is another question...